properties/src/main/java/gov/cms/ab2d/properties/dto/PropertiesDTO.java

```diff
@@ -0,0 +1,20 @@
+package gov.cms.ab2d.properties.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+
+@Data
+@AllArgsConstructor
+public class PropertiesDTO {
```

Comment:
Unless these classes are radically different, try reverting the delete/add and simply do a move refactoring.  Then GIT will preserve history and simply show the file as being moved.

Reply:
This applies to all the other delete/add operations.

Reply:
They are enough different for such a small file. I did a move, not an add or delete, it just interpreted as such.